functions/zoom-downloader.py

```diff
@@ -146,10 +123,12 @@ def stream_file_to_s3(file, metadata):
                                      MultipartUpload=part_info)
         print("Completed multipart upload of {}.".format(filename))
     except Exception as e:
-        logger.exception("Something went wrong with upload of {}".format(filename))
+        logger.exception("Something went wrong with upload of {}:{}".format(filename, e))
```

Comment:
the `.exception()` logger method should automatically include the exception info.